### PR TITLE
Convert afc commands to rzshell

### PR DESCRIPTION
**Pull request by p371jah**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

- `afcR` was removed (untested)
- `afck` was removed (dup of `afclk`)
- `afcf` was removed (dup of `afs`)
- new IPI `rz_core_analysis_cc_print` (is also used in `rz_core_types_calling_conventions_print`)
- fixed infinite loop bug in `rz_analysis_function_get_json`

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

File: test/db/cmd/cmd_af [hunk 1]
```diff
@@ -269,32 +269,30 @@ size: 12
...
-afcf
-afcfj
+afs
+afsj
 EOF
 EXPECT=<<EOF
-fcn.00013c70 (int64_t arg1, int64_t arg2, int64_t arg3);
-[{"name":"fcn.00013c70","args":[{"name":"arg1","type":"int64_t"},{"name":"arg2","type":"int64_t"},{"name":"arg3","type":"int64_t"}],"count":3}]
+void fcn.00013c70(int64_t arg1, int64_t arg2, int64_t arg3);
+{"name":"fcn.00013c70","noreturn":false,"ret":"void","cc":"amd64","args":[]}
```
Comment (p371jah) [inline test/db/cmd/cmd_af:295]: See #2561

File: librz/core/cmd/cmd_analysis.c [hunk 2]
```diff
@@ -7403,6 +7240,56 @@ RZ_IPI RzCmdStatus rz_analysis_functions_merge_handler(RzCore *core, int argc, c
 	return RZ_CMD_STATUS_OK;
 }
 
+RZ_IPI RzCmdStatus rz_analysis_function_cc_set_get_handler(RzCore *core, int argc, const char **argv) {
+	RzAnalysisFunction *fcn = analysis_get_function_in(core->analysis, core->offset);
+	if (!fcn) {
+		return RZ_CMD_STATUS_ERROR;
```
Comment (XVilka) [inline librz/core/cmd/cmd_analysis.c:7246]: Probably worth showing the error "no function at this offset" or something

**Review by XVilka — APPROVED** [review]
LGTM

File: librz/core/cmd/cmd_analysis.c (hunk 2 above)
Comment (p371jah) [reply librz/core/cmd/cmd_analysis.c:7246]: `analysis_get_function_in` does it

**Review by XVilka — APPROVED** [review]

**Comment by ITAYC0HEN** [comment]
Caused a regression on Cutter 

https://github.com/rizinorg/cutter/blob/6f61d0d7d84bef37505eacd31d85825701dc96b0/src/widgets/GraphWidget.cpp#L85-L94

**Comment by wargio** [comment]
https://github.com/rizinorg/cutter/pull/2940

**Pull request merged**